nixos/tests/installer.nix

```diff
@@ -312,61 +331,61 @@ let
 
     makeLuksRootTest = name: luksFormatOpts: makeInstallerTest name
       { createPartitions = ''
-          $machine->succeed(
-            "flock /dev/vda parted --script /dev/vda -- mklabel msdos"
-            . " mkpart primary ext2 1M 50MB" # /boot
-            . " mkpart primary linux-swap 50M 1024M"
-            . " mkpart primary 1024M -1s", # LUKS
-            "udevadm settle",
-            "mkswap /dev/vda2 -L swap",
-            "swapon -L swap",
-            "modprobe dm_mod dm_crypt",
-            "echo -n supersecret | cryptsetup luksFormat ${luksFormatOpts} -q /dev/vda3 -",
-            "echo -n supersecret | cryptsetup luksOpen --key-file - /dev/vda3 cryptroot",
-            "mkfs.ext3 -L nixos /dev/mapper/cryptroot",
-            "mount LABEL=nixos /mnt",
-            "mkfs.ext3 -L boot /dev/vda1",
-            "mkdir -p /mnt/boot",
-            "mount LABEL=boot /mnt/boot",
-          );
+          machine.succeed(
+              "flock /dev/vda parted --script /dev/vda -- mklabel msdos"
+              " mkpart primary ext2 1M 50MB"  # /boot
+              " mkpart primary linux-swap 50M 1024M"
+              " mkpart primary 1024M -1s",  # LUKS
```

Comment:
it's not necessary, but might be helpful while reading to prefix those multiline-lines with `+`

Reply:
good point. might try that